docs/helpers/AppPinManager.md

```diff
@@ -0,0 +1,58 @@
+---
+title: AppPinManager helper
+author: rvinothrajendran
+ms.date: 08/11/2017
```

Comment:
Make sure the date is in the format mm/dd/year

Reply:
@nmetulev all changes has done , including sample application 

one clarification @nmetulev 

Rebuild the application , In output folder  \Microsoft.Toolkit.Uwp.SampleApp\bin\x86\Debug\AppX\SamplePages\AppPinManager\AppPinManagerHelperCode.bind file is not created automatically , anything I missed out ?

Thanks & Regards,
Vinoth.